src/components/GatedResourceLayout/index.tsx

```diff
@@ -58,25 +62,21 @@ export const GatedResourceLayout: FunctionComponent<Props> = ({
     const isWebinarPg = useRouter().pathname.split('/').slice(1)[0] === 'webinars'
     const isGuidePg = useRouter().pathname.split('/').slice(1)[0] === 'guides'
 
-    const hubSpotConfig: HubSpotForm = {
+    const hubSpotConfig: HubSpotForm | null = {
         portalId: '2762526',
-        formId: form.formId,
+        formId: form?.formId ?? '',
```

Comment:
The `formId` is always required if you're using the HubSpot hook so I don't think we should allow it to ever be optional. I see why you're allowing this now that I've looked at the rest of the code - more on this in further comments.

```suggestion
        formId: form.formId,
```